pages/migration/delegator.tsx

```diff
@@ -39,6 +77,68 @@ export default function DelegateMigrationPage(): React.ReactElement {
               One or more of your MakerDAO delegate&lsquo;s contracts are expiring.
             </Text>
           )}
+
+          <Box>
+            <Text as="p" variant="body">
+              Maker delegate contracts expire after 1 year.
+            </Text>
+            <Text>
+              Please migrate your MKR by undelegating from the expiring/expired contracts and redelegating to
+              the new contracts.
+            </Text>
+          </Box>
+
+          <Box>
+            <Text>
+              Please find below any expiring/expired delegate contracts you delegated MKR to, and requires
+              migration.
+            </Text>
+          </Box>
+
+          <Box>
+            <Box>
+              <Text as="h1">Expired/about to expire contracts you delegated MKR to</Text>
+              <Text as="p" variant="body">
+                Please undelegate your MKR from old contracts, one by one. Might require aproval transactions.
+              </Text>
+            </Box>
+
+            {delegatesThatAreAboutToExpiry.length > 0 && (
+              <Box>
+                {delegatesThatAreAboutToExpiry.map(delegate => {
+                  return (
+                    <Box key={`delegated-about-to-expiry-${delegate.address}`} sx={{ mb: 3 }}>
+                      <DelegateExpirationOverviewCard delegate={delegate} />
+                    </Box>
+                  );
+                })}
+              </Box>
+            )}
+            {delegatesThatAreAboutToExpiry.length === 0 && (
+              <Box>
+                <Text>There are no delegates that are about to expiry.</Text>
+              </Box>
+            )}
+          </Box>
+
+          <Box>
+            <Box>
+              <Text as="h1">Renewed contracts by your previous delegates</Text>
+              <Text as="p" variant="body">
+                Please delegate your MKR to the new contracts, one by one. Might require aproval transactions.
```

Comment:
I know this is from the design but we know for a fact that approval transactions will be required, so we can update this copy. I think Deniz will review it all though.